shared/modules/module.go

```diff
@@ -21,6 +24,21 @@ type InterruptableModule interface {
 
 type InitializableModule interface {
 	GetModuleName() string
-	InitConfig(pathToConfigJSON string) (IConfig, error)
-	InitGenesis(pathToGenesisJSON string) (IGenesis, error)
+	Create(runtime RuntimeMgr) (Module, error)
```

Comment:
Non-Blocker:

The granularity here is getting a bit sticky for me. Consider mental load vs. verbosity

Reply:
I see. If it sounds like over-optimization or premature optimization let's consider the fact that soon we'll have modules that are very simple and only implement a subset of the Module interface. Logging for example, which is coming soon. Keybase as well I guess.

Maybe my .Net background surfaces here. I am used to very small and targeted interfaces.

I am open to suggestions.